data/crac-file/crac-file-xlsx-api/src/main/java/com/farao_community/farao/data/crac_file/xlsx/service/CracTools.java

```diff
@@ -47,7 +47,7 @@ public static String getOrderCodeElementName(ElementDescriptionMode type, String
         }
         if (null != type) {
```

Comment:
Je pense qu'il serait mieux de retirer cette ligne, car ça augmente la complexité du code et de faire un **if(null != type && type.equals(ElementDescriptionMode.ORDER_CODE)**

Reply:
it's not useful to delete this line because it used many times